types/benches/bytesrepr_bench.rs

```diff
@@ -437,6 +446,300 @@ fn deserialize_u512(b: &mut Bencher) {
     b.iter(|| U512::from_bytes(black_box(&num_u512_bytes)))
 }
 
+fn generate_account(associated_keys_len: u8, named_keys_len: u8) -> Account {
+    let account_hash = AccountHash::default();
+    let named_keys: NamedKeys = {
+        let mut tmp = BTreeMap::new();
+        (0..named_keys_len).for_each(|i| {
+            let res = tmp.insert(
+                std::str::from_utf8(&[i]).unwrap().to_string(),
+                Key::Account(AccountHash::default()),
+            );
+            assert!(res.is_none());
+        });
+        tmp
+    };
+    let main_purse = URef::default();
+    let associated_keys = {
+        let mut tmp = AssociatedKeys::new(AccountHash::default(), Weight::new(1));
+        (1..associated_keys_len).for_each(|i| {
+            tmp.add_key(
+                AccountHash::new([i; casper_types::account::ACCOUNT_HASH_LENGTH]),
+                Weight::new(1),
+            )
+            .unwrap()
+        });
+        tmp
+    };
+    let action_thresholds = ActionThresholds::default();
+    Account::new(
+        account_hash,
+        named_keys,
+        main_purse,
+        associated_keys,
+        action_thresholds,
+    )
+}
+
+fn serialize_account(b: &mut Bencher) {
+    let account = generate_account(10, 10);
+    b.iter(|| ToBytes::to_bytes(black_box(&account)));
+}
+
+fn deserialize_account(b: &mut Bencher) {
+    let account = generate_account(10, 10);
+    let account_bytes = Account::to_bytes(&account).unwrap();
+    b.iter(|| Account::from_bytes(black_box(&account_bytes)).unwrap());
+}
+
+fn serialize_contract(b: &mut Bencher) {
+    let contract = generate_contract(10, 10);
+    b.iter(|| ToBytes::to_bytes(black_box(&contract)));
+}
+
+fn deserialize_contract(b: &mut Bencher) {
+    let contract = generate_contract(10, 10);
+    let contract_bytes = Contract::to_bytes(&contract).unwrap();
+    b.iter(|| Contract::from_bytes(black_box(&contract_bytes)).unwrap());
+}
+
+fn generate_contract(named_keys_len: u8, entry_points_len: u8) -> Contract {
+    let named_keys: NamedKeys = {
+        let mut tmp = BTreeMap::new();
+        (0..named_keys_len).for_each(|i| {
+            let res = tmp.insert(
+                std::str::from_utf8(&[i]).unwrap().to_string(),
```

Comment:
```suggestion
                i.to_string(),
```
or `format!("named-key-{}", i)`

Reply:
Done in [`b07c708` (#2389)](https://github.com/casper-network/casper-node/pull/2389/commits/b07c70853e7d9447e20fd5a17ad635a5b8606ad5).